### DISCUSSION TITLE
Where to check for detailed error msg for `error running bundle_dmg.sh`?

### DISCUSSION THREAD

**Discussion by washanhanzi** [tauri-apps/tauri#6598] [Q&A] [answered]
I want to create a tauri app with qwik. It's alright with development, but when I build the app, the app show a blank screen. Then I enabled SSG for qwik, and tauri got a build error `error running bundle_dmg.sh` with no detailed information.

I am  on m2 macbook.

**Comment by FabianLars** [comment, marked as answer]
You can add the verbose flag when building your app, for example: `yarn tauri build --verbose` - if you use npm it needs an extra `--` like this `npm run tauri build -- --verbose`. Also try removing your target dir so that you get a fresh build.

**Comment by washanhanzi** [comment]
I followed [this](https://github.com/tauri-apps/tauri/discussions/4683). It seems ok now. But still got blank screen on build with qwik.

**Reply by FabianLars** [reply]
You can add the `"devtools"` feature flag to the tauri dependency in Cargo.toml or create a debug build (`tauri build --debug` output in target/debug/) to get access to the devtools console which you can then check for errors.

**Reply by washanhanzi** [reply]
hmm, I tried this. And suddenly it works. I don't what happened. But i got an error from the dev tools `TypeError: serviceWorker.register() must be called with a script URL whose protocol is either HTTP or HTTP`. Really thank you for your time.

And by the way, if I include `devtools` in `cargo.toml`, it does't matter whether I include the `--debug` flag, I always got a build with  `devtool` enabled. Maybe I misunderstand the usage of `--debug` flag.

**Reply by FabianLars** [reply]
yeah so the devtools are always enabled for debug builds but never for release builds, unless you add the feature flag which will activate it for release builds too.

**Reply by washanhanzi** [reply]
What I mean is if I add `devtools` in `cargo.toml`, even without the `--debug` flag, the build is still with `devtool` enabled.

**Reply by FabianLars** [reply]
Yes i understand, and tried to explain why 😅 What you're describing is the _intended_ behavior. I listed 2 _different_ ways to get access to the devtools, i didn't mean to tell you that you have to do both things at the same time.

**Reply by washanhanzi** [reply]
Got it, thank you again.
